mpp/include/mpp_domains_misc.inc

```diff
@@ -419,6 +419,13 @@ end subroutine init_nonblock_type
       field2(isd:ied, jsd:jed) = field_in(:,:)
     endif
 
+!  allocate field1 on pelist1
```

Comment:
@laurenchilutti, I just wanted to make sure that the changes in mpp are correct here.